src/main.rs

```diff
@@ -17,6 +17,9 @@ use diesel::pg::PgConnection;
 use std::env;
 
 use ring::{digest, rand, pbkdf2};
```

Comment:
I'm getting an unused import when compiling. Do you need to include `rand` this if you're using `ring::rand::SecureRandom` on line 21?